css/cssom-view/matchMedia.html

```diff
@@ -0,0 +1,92 @@
+<!DOCTYPE html>
+<meta charset="utf-8">
+<meta name="flags" content="dom">
+<title>CSS Test: CSSOM View matchMedia and MediaQueryList</title>
+<link rel="author" title="Rune Lillesveen" href="mailto:rune@opera.com">
+<link rel="help" href="http://www.w3.org/TR/cssom-view/#dom-window-matchmedia">
+<link rel="help" href="http://www.w3.org/TR/cssom-view/#the-mediaquerylist-interface">
+<link rel="help" href="http://www.w3.org/TR/cssom-1/#serializing-media-queries">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="resources/matchMedia.js"></script>
+<div id="log"></div>
+<script>
+"use strict";
+
+test(() => {
+    assert_equals(
+        typeof window.matchMedia,
+        "function",
+        "FATAL ERROR: The window.matchMedia function is not present. The rest of the testsuite will fail to run."
+    );
+}, "window.matchMedia is a function");
+
+test(() => {
+    const mql = window.matchMedia("all");
+    assert_true(mql instanceof MediaQueryList);
```

Comment:
Looks like [`idlharness.html#L28`](https://github.com/web-platform-tests/wpt/blob/master/css/cssom-view/idlharness.html#L28) does exactly this and it also pretty much covers the next 2 tests.